grades/tasks.py

```diff
@@ -0,0 +1,125 @@
+"""
+Tasks for the course app
+"""
+import logging
+
+from celery import group
+from celery.result import GroupResult
+from django.core.cache import cache
+
+from courses.models import CourseRun
+from grades import api
+from grades.models import CourseRunGradingStatus
+from micromasters.celery import async
+from micromasters.utils import chunks
+
+
+CACHE_ID_BASE_STR = "freeze_grade_{0}"
+
+log = logging.getLogger(__name__)
+
+
+@async.task
+def find_course_runs_for_grade_freeze():
```

Comment:
Can you rename this task? `find_course_runs_for_grade_freeze` seem like a task that just collects information, but this task also freezes grades. Maybe `find_course_runs_and_freeze_grades`?